subsys/bluetooth/mesh/sensor_srv.c

```diff
@@ -394,10 +399,8 @@ static int handle_series_get(struct bt_mesh_model *model, struct bt_mesh_msg_ctx
 		goto respond;
 	}
 
-	struct bt_mesh_sensor_column range;
-	bool ranged = (buf->len != 0);
-
 	if (buf->len == col_format->size * 2) {
+		struct bt_mesh_sensor_column range;
```

Comment:
Seems this changs previous behavior. `range` kept value between `if` condition later `for` loop.
Now `range` value is lost between them. 

I suppose Coverity fixes shouldn't change behavior.

Reply:
I will ignore the coverity warning. This is way I left ranged now, but the reason I took both. But I moved range back.
CID 113383 (#7 of 7): Branch past initialization (PW.BRANCH_PAST_INITIALIZATION)
1. branch_past_initialization: transfer of control bypasses initialization of:
 	2. name_at_decl_position: variable "ranged" (declared at line 398)